openstates/az/action_utils.py

```diff
@@ -1,212 +1,125 @@
-    'DP/PFC W/FL': 'do pass and proper for consideration with recommendation for a floor amendment',
-    'DP/PFCA': 'do pass and proper for consideration amended',
-    # favorable and amended from here down #
-    'DPA': 'do pass amended',
-    'DPA CORRECTED': 'do pass amended corrected',
-    'DPA ON RECON': 'do pass amended on reconsideration',
-    'DPA ON REREFER': 'do pass amended on rereferral',
-    'DPA/PFC': 'do pass amended and proper for consideration',
-    'DPA/PFC W/FL': 'do pass amended and proper for consideration with recommendation for a floor amendment',
-    'DPA/PFCA': 'do pass amended and proper for consideration amended',
-    'DPA/PFCA W/FL': 'do pass amended and proper for consideration with recommendation for a floor amendment',
-    # strike everything is like amended in the nature of a substitue
-    'DPA/SE': 'do pass amended/strike-everything',
-    'DPA/SE CORRECTED': 'do pass amended/strike everything corrected',
-    'DPA/SE ON RECON': 'do pass amended/strike everything on reconsideration',
-    'DPA/SE ON REREF': 'do pass amended/strike everything on rereferral',
-    # failed #
-    'FAILED': 'failed to pass',
-    'FAILED BY S/V 0': 'failed by standing vote', # does this mean there is no vote?
-    'FAILED ON RECON': 'failed on reconsideration', # after a succesful motion to reconsider
-    # in the house a bill is first read and assigned to committee #
-    'FIRST': 'First Reading',
-    # amendment amended??? or just another amendment? #
-    'FURTHER AMENDED': 'further amended',
-
-    'HELD': 'held',
-    'HELD 1 WK': 'held one week',
-    'HELD INDEF': 'held indefinitely',
-    'HELD ON RECON': 'held on reconsideration',
-
-    'None': 'No Action',
-
-    'NOT CONCUR': 'rec not concur',
-    # not read?
-    'NOT HEARD': 'not heard',
-
-    'NOT IN ORDER': 'not in order',
-
-    'PASSED': 'Passed',
-    # rules committee action #
-    'PFC': 'proper for consideration',
-
-    'PFC W/FL': 'proper for consideration with recommendation for a floor amendment',
-    'PFCA': 'proper for consideration amended',
-    'PFCA W/FL': 'proper for consideration amended with recommendation for a floor amendment',
-
-    'POSTPONE INDEFI': 'postponed indefinitely',
-    # vote actions
-    'REC REREF TO COM': 'recommend rereferral to committee',
-    'RECOMMIT TO COM': 'recommit to committee',
-    # not sure when this would take place? I would like to see it in action
-    'REMOVAL REQ': 'removal request from Rules Committee',
-    # committee:refered
-    'REREF GOVOP': 'rereferred to GovOp',
-    'REREF JUD': 'rereferred to Judiciary',
-    'REREF WM': 'rereferred to Ways & Means',
-
-    'RET FOR CON': 'returned for consideration',
-
-    'RET ON CAL': 'retained on the Calendar',
-    'RETAINED': 'retained',
-
-    'RULE 8J PROPER': 'proper legislation and deemed not derogatory or insulting',
-
-    'S/C': 'subcommittee',
-    'S/C REPORTED': 'subcommittee reported',
-
-    'W/D': 'withdrawn',
-}
+action_map = {
```

Comment:
@jamesturk this chunk is my biggest concern, I tried to create a mapping between the various JSON keys the state provided, and name/actions. 

Not sure this approach is smart, so open to a refactor here if you've got better ideas.